packages/manager/src/components/ErrorState/ErrorState.tsx

```diff
@@ -28,10 +29,19 @@ export interface ErrorStateProps {
    * Styles applied to the error text
    */
   typographySx?: SxProps<Theme>;
+  /**
+   * Optional Action Button: Text
+   */
+  actionButtonText?: string;
+  /**
+   * Optional Action Button: Click handler
+   */
+  onActionButtonClick?: ()=>void;
```

Comment:
We can make a separate interface to group the two props together and make it optional

```typescript
interface ActionButtonProps {
  onClick: () => void;
  text: string;
}
```

```suggestion
  actionButtonProps?: ActionButtonProps;
```

Reply:
Great idea! I have pushed an update.